ezgui/src/backend_glow.rs

```diff
@@ -244,9 +182,15 @@ impl Drop for Buffer {
     }
 }
 
+#[cfg(feature = "wasm-backend")]
+type WindowAdapter = crate::backend_glow_wasm::WindowAdapter;
+
+#[cfg(feature = "glow-backend")]
+type WindowAdapter = crate::backend_glow_native::WindowAdapter;
+
 pub struct PrerenderInnards {
     gl: Rc<glow::Context>,
-    windowed_context: glutin::WindowedContext<glutin::PossiblyCurrent>,
+    window_adapter: WindowAdapter,
```

Comment:
The new `WindowAdapter` encapsulates the difference between wasm vs native window behavior.